src/components/display/Alert.jsx

```diff
@@ -0,0 +1,55 @@
+import React, {useState, useEffect} from 'react';
+import PropTypes from 'prop-types';
+import {Box, Snackbar} from '@material-ui/core';
+import {Alert as MuiAlert} from '@material-ui/lab';
+
+/**
+ * Alert component
+ */
+const Alert = ({id = 'alert', message = null}) => {
```

Comment:
💡 

What about renaming the props when extracting them?

```js
const Alert = (props) => {
  const { id, message: initialMessage } = props;
  const [message, setMessage] = useState(initialMessage);
  ...


Alert.defaultProps = {
  id: 'alert',
  message: null
};
```

Reply:
Thanks, it's better !